pages/index.js

```diff
@@ -64,39 +79,54 @@ export default function Home() {
       </Row>
       </div>
 
-      <div className={`${styles.sectionColorLight} `}>
+      {/* Events Section */}
+      <div className={"sectionAlt"}>
         <Row className="justify-content-center">
           <Col lg={12} sm={12}>
             <h2 className={`${styles.sectionTitle}`}>Our Events</h2>
             <p className={`${styles.sectionDesc}`}>We aim to host events that will better ICS Students academically, professionally, and socially.</p>
-            <p className={`${styles.sectionLink}`}><a href="/events">View Events ></a></p>
+            <p className={`${styles.sectionDesc}`}>Here are some moments from our previous events!</p>
           </Col>
         </Row>
-        <div className={`${styles.elementList}`}>
-              {EventsData["past"].slice(0,3).map(past => <Event {...past} key={past.id}/>)}
+        <div style={{width:"400px"}}>
+          <Carousel controls={false} indicators={false}>
+          {EventsData["past"].map(past => (
+            <Carousel.Item>
+              <img
+                className="d-block w-100"
+                src={past.src}
+                alt={past.title}
+                height="400px"
+              />
+            </Carousel.Item>
+          ))}
+          </Carousel>
         </div>
+        <p className={`${styles.sectionLink}`}><a href="/events"><br></br>View All Events ></a></p>
       </div>
 
-      <div className={`${styles.section}`}>
+      {/* Projects Section */}
+      <div className={"section"}>
         <Row className="justify-content-center">
             <Col lg={12} sm={12}>
             <h2 className={`${styles.sectionTitle}`}>Projects</h2>
             <p className={`${styles.sectionDesc}`}>
               Our open source projects are to help students succeed at finding, planning, executing, and managing technology projects. It is our goal to teach students useful technical knowledge outside of class. All students are welcome to contribute.
             </p>
-            <p className={`${styles.sectionLink}`}><a href="/projects">Learn More ></a></p>
+            <p><a href="/projects">Learn More ></a></p>
```

Comment:
Probably don't need the `<p>` wrapping the the `<a>` but idk if that messes up styling
```suggestion
            <a href="/projects">Learn More ></a>
```

Reply:
Yeah the anchor tag gets the style of the p tag. I'm not too sure about the style quality, but by nesting it I save some CSS lines.